docs/config.rst

```diff
@@ -1046,6 +1046,14 @@ Marks
 
    See also: :attr:`LP_ENABLE_HG` and :attr:`LP_HG_COMMAND`.
 
+.. attribute:: LP_MARK_JOBS_SEPARATOR
+   :type: string
+   :value: "/"
+
+   Mark used to separate elements of :attr:`LP_JOBS`.
+
+   See also :attr:`LP_ENABLE_JOBS`.
+
```

Comment:
Needs a version added note.

Reply:
I thought I had it on such a simple one… :man_facepalming: